index.js

```diff
@@ -1,9 +1,17 @@
 function calculate(numberOne, operation, numberTwo) {
+  if (isNaN(numberOne)) {
```

Comment:
Nice use of `isNaN`
I believe you swa someone using it this week and you asked about it. Quick turnaround on using it. Well done.

Reply:
Thanks!